src/Symfony/Component/Form/FormBuilder.php

```diff
@@ -66,10 +66,6 @@ public function add($child, $type = null, array $options = [])
             throw new UnexpectedTypeException($child, 'string, integer or Symfony\Component\Form\FormBuilderInterface');
         }
 
-        if (null !== $type && !\is_string($type) && !$type instanceof FormTypeInterface) {
```

Comment:
👍 but we should fix this in lower branches because FormTypeInterface is indeed not a valid value here

Reply:
#32934